specification/data-semantic-conventions.md

```diff
@@ -279,33 +284,67 @@ but due to `http.scheme`, `http.host` and `http.target` being set, it would be r
 As explained above, these separate values are preferred but if for some reason the URL is available but the other values are not,
 URL can replace `http.scheme`, `http.host` and `http.target`.
 
-## Databases client calls
+## Databases (SQL and NoSQL) client calls
 
 > 🚧 WORK IN PROGRESS
 
 For database client call the `SpanKind` MUST be `Client`.
 
 Span `name` should be set to low cardinality value representing the statement
-executed on the database. It may be stored procedure name (without argument), sql
+executed on the database. It may be stored procedure name (without argument), SQL
 statement without variable arguments, etc. When it's impossible to get any
 meaningful representation of the span `name`, it can be populated using the same
 value as `db.instance`.
 
-Note, Redis, Cassandra, HBase and other storage systems may reuse the same
-attribute names.
+### Connection-level attributes
+
+These attributes will usually be the same for all operations performed over the same database connection
+although some database systems may allow to switch to a different `db.user` within the same connection
+and other database systems may not even have the concept of a connection.
 
 | Attribute name | Notes and examples                                           | Required? |
 | :------------- | :----------------------------------------------------------- | --------- |
 | `component` | Always the string `"db"` | Yes       |
-| `db.tech`      | The lower-case database category, e.g. `"sql"`, `"cassandra"`, `"hbase"`, or `"redis"`, optionally followed by a dot `"."` and the more specific technology name (e.g. `"sql.MySQL"`). | Yes       |
-| `db.instance`  | Database instance name. E.g., In Java, if the jdbc.url=`"jdbc:mysql://db.example.com:3306/customers"`, the instance name is `"customers"`. | Yes     |
-| `db.statement` | A database statement for the given database type. Note, that the value may be sanitized to exclude sensitive information. E.g., for `db.type="sql"`, `"SELECT * FROM wuser_table"`; for `db.type="redis"`, `"SET mykey 'WuValue'"`. | Yes       |
-| `db.url` | The connection string used to connect to the database | Yes       |
-| `db.resultcount` | An integer specifying the number of results returned. | No       |
-| `db.roundtripcount` | An integer specifying the number of network roundtrips while executing the request. | No       |
+| `db.type` | The lower-case database category, e.g. `"sql"`, `"cassandra"`, `"hbase"`, or `"redis"`. | Yes       |
+| `db.tech` | Name of the database technology used, e.g. "SQL Server" or "MySQL". | No     |
```

Comment:
`tech` and `type` feels redundant.
I would remove the `type` attribute altogether and have a descriptive name in `tech`.

The actual "tech" can be inferred from:
* the existence of the `sql` property, indicating that it is a SQL database.
* a "tech namespace", specifying additional attributes specific to a DB. E.g.:
  *  `db.mongodb.query`,
  * `db.oracle.<something_oracle_specific>`